blocks/social/social.js

```diff
@@ -0,0 +1,59 @@
+import {
+  div, span, img, a,
+} from '../../scripts/dom-builder.js';
+
+const SOCIAL_CONFIGS = [
```

Comment:
[See this conversation](https://adobe-dx-support.slack.com/archives/C06H8UNQ3RS/p1707735741147009)